pkgs/top-level/python-packages.nix

```diff
@@ -3392,6 +3392,24 @@ let
     };
   };
 
+  gateone = buildPythonPackage rec {
```

Comment:
I don't have the CPU cycles to test this (nor does Travis, apparently), but the odds of this package needing something like `disabled = ! isPy3k` (random example!) is probably high.